README.adoc

```diff
@@ -1,22 +1,177 @@
-= Collaborative exams
-:sectanchors:
-
-image:https://travis-ci.org/badga/Collaborative-exams.svg?branch=master["Build Status", link="https://travis-ci.org/badga/Collaborative-exams"]
-
-A Java project developed during the https://github.com/oliviercailloux/java-course[Java course] taught at Université Paris-Dauphine (in http://www.mido.dauphine.fr/[MIDO] M2 MIAGE). The code is jointly copyrighted by Olivier Cailloux and the following students:
-
-* Jeff Azzam
-* Sid Ahmed Imloul
-* Sofiath Lanignan
-* Lamine Tamendjari
-* Omar Yaghi
-* Abdelmadjid Yous
-
-The code is hereby published under the MIT License, with their permission.
-
-== Original description
-Ce projet permettra à des enseignants de créer des ensembles de questions et réponses. Ces questions peuvent être réutilisées par d’autres enseignants et combinées pour créer des questionnaires ou des sondages. Les étudiants pourront se servir de ces ensembles de questions pour s’entrainer.
-
-=== Retrouvez notre application en suivant le lien ci dessous
-http://collaborative-exams.eu-gb.mybluemix.net/
-
+= Première itération 
+
+*Amine* 
+
+* Création d'un nouvel auteur en renseignant un formulaire
+* Authentification d'un auteur à l'aide de ses identifiants
+* Création des Servlets associés aux auteurs
+* Création d'une vue permettant l'authentification
+* Création d'une vue permettant l'insertion des données lors de la création d’un nouvel auteur
+* Insérer les nouveaux auteurs dans la BDD
+* Gérer en JSON et XML les auteurs 
+

```

Comment:
À la place de ces fct, je vous conseille fortement de commencer par revoir et consolider le code de Question et ce qui l’entoure (persistence, service…). Il y a fort à faire.

Il faut créer une hiérarchie de questions qui permettra de les gérer facilement, en parallèle de l’entité elle-même. Créer une interface IQuestion, et des types concrets FreeQuestion, FMCQuestion (Multiple Choice), qui a une méthode getPropositions (remplaçant getAnswers actuelle), YesNoQuestion qui étend MCQuestion, TrueFalseQuestion qui étend MCQuestion, etc.

Il faut aussi écrire les tests unitaires.

Reply:
Parfait  ! Je vais m'attaquer à ces fonctionnalités dans un premier temps et effectuer les tests unitaires nécessaires 
Dois-je modifier mon cahier des charges ? sachant que j'ai déjà réalisé la plupart de mes fonctionnalités de mon ancien cahier des charges à savoir, la gestion des auteurs.

Reply:
@oliviercailloux J'ai modifié le cahier des charges de ma première itération.

Je travaille actuellement sur les questions comme vous le souhaitiez.

Prière de ma valider cette partie.

Reply:
Je vous prie de copier mon commentaire du 17 janvier qui détaille les interfaces (IQuestion, FreeQuestion, etc.) dans votre cahier des charges, en plus de la description actuelle de Question, pour qu’on puisse retrouver facilement ces informations.